Packs/Sixgill-Darkfeed/ReleaseNotes/2_0_0.md

```diff
@@ -2,3 +2,64 @@
 #### Integrations
 ##### New: Cybersixgill DVE Feed Threat Intelligence
 - Leverage the power of Sixgill to supercharge Cortex XSOAR with real-time Threat Intelligence indicators. Get CVE Feed straight into the XSOAR platform. (Available from Cortex XSOAR 5.5.0).
+
+#### Indicator Fields
+##### New: CVSS 2.0 Severity
+- New field
+
```

Comment:
@BenSterenson I would add here the description of each field.

Reply:
@reutshal done